packages/mux-player/src/media-theme-mux/media-theme-mux.ts

```diff
@@ -138,8 +138,8 @@ export default class MediaThemeMux extends MediaTheme {
           <slot name="media" slot="media"></slot>
           <media-poster-image
             slot="poster"
-            src="${props.poster}"
-            placeholder-src="${props.placeholder}"
+            src="${props.poster ?? false}"
+            placeholder-src="${props.placeholder ?? false}"
```

Comment:
these are necessary because `poster` and `placeholder` props may not always be booleans and that might cause issues when toggling the values?

Reply:
it's our template logic that works like this. when setting an explicit `false` it will remove the attribute.
otherwise it will still set an attribute value with null or undefined